src/examples/rest/src/backend/index.ts

```diff
@@ -16,7 +14,7 @@ export enum Roles {
 const graphweaver = new Graphweaver<AuthorizationContext>({
 	resolvers,
 	apolloServerOptions: {
-		plugins: [localAuthApolloPlugin(addUserToContext), connectToDatabase(myConnection)],
```

Comment:
we also need to remove this plugin from the `cli init` template and the introspection template.